src/docker-push-to-ecr.sh

```diff
@@ -101,7 +101,11 @@ function main() {
   AWS_ACCESS_KEY_ID=$Key AWS_SECRET_ACCESS_KEY=$Secret aws ecr get-login --no-include-email --region us-east-1 | /bin/bash
 
   echo "Building, tagging and pushing version $Version$Suffix"
-  docker build "$DockerArgs" -t "$Repo:latest$Suffix" -t "$Repo:$Version$Suffix" "$Dockerfile"
+  if [ -z "$DockerArgs" ]; then
```

Comment:
An alternate solution would be to not quote the `DockerArgs` variable, but the linter yells about that:

```
In docker-push-to-ecr.sh line 105:
    docker build $DockerArgs -t "$Repo:latest$Suffix" -t "$Repo:$Version$Suffix" "$Dockerfile"
                 ^---------^ SC2086: Double quote to prevent globbing and word splitting.

Did you mean:
    docker build "$DockerArgs" -t "$Repo:latest$Suffix" -t "$Repo:$Version$Suffix" "$Dockerfile"
```